src/page-footer.scss

```diff
@@ -0,0 +1,126 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-page-footer;
+
+$fd-page-footer-sizes: (
+  "xl": ("padding": 3rem),
+  "lg": ("padding": 2rem),
+  "md": ("padding": 1rem),
+  "sm": ("padding": 0.5rem)
+);
+
+.#{$block} {
+  @include fd-reset();
+
+  $fd-footer-padding-top: 0.75rem !default;
+  $fd-footer-margin-right: 1.5rem !default;
+  $fd-footer-link-padding-top: 0.25rem !default;
+
+  @mixin fd-truncate-text {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
```

Comment:
there's already a mixin for it. Please check the mixibs file

Reply:
multiline-truncate() i see this mixin other than this there is no mixin available and this mixin add's unwanted margin on right. and there is no '...' at the end of the overflow text.
<img width="522" alt="Screenshot 2021-04-14 at 11 47 41 AM" src="https://user-images.githubusercontent.com/53534379/114663016-43121800-9d17-11eb-869c-3adcde026fb8.png">